### DISCUSSION TITLE
Mavlink UDP C++ library that anyone can use

### DISCUSSION THREAD

**Discussion by eMrazSVK** [mavlink-router/mavlink-router#443] [Show and tell]
Hello guys,

So I've finally finished C++ Mavlink interface. We use it with mavlink-router for many of Airvolute applications and it works great. I know that lot of people struggle with receiving/sending messaging directly from C++. I think this could complete the ecosystem around Mavlink, together with your mavlink-router.

- https://gitlab.com/airvolute_oss/dronecore.os/mavlink-udp-interface -> Main Mavlink C++ library
- https://gitlab.com/airvolute_oss/dronecore.os/mavlink-udp-comm-template -> Template application showcasing the functionality

**Comment by jbeyerstedt** [comment]
I would rather put this in the "show and tell" section of the GitHub discussions.

**Reply by eMrazSVK** [reply]
Okay, so we just leave it here so people can learn about it, or do you prefer some specific form of this?
